TangoRosStreamer/app/src/main/java/eu/intermodalics/tango_ros_streamer/activities/AboutActivity.java

```diff
@@ -48,7 +48,7 @@ protected void onCreate(Bundle savedInstanceState) {
         TextView textViewVersion = (TextView) findViewById(R.id.text_version);
         try {
             PackageInfo packageInfo = getPackageManager().getPackageInfo(getPackageName(), 0);
-            textViewVersion.setText(packageInfo.versionName + "-" + packageInfo.versionCode);
+            textViewVersion.setText(packageInfo.versionName);
```

Comment:
It was actually an error, we were using the version code as the patch version, which is wrong. The patch version should be included in the version name and is totally independent from the version code.